source/index.txt

```diff
@@ -31,15 +31,14 @@ MongoDB Java Driver
    /legacy
    View the Source <https://github.com/mongodb/mongo-java-driver>
 
-
 Introduction
 ------------
 
 Welcome to the documentation site for the Java Driver, the official
-MongoDB driver for synchronous Java applications. Download it using
-`Maven <https://maven.apache.org/>`__ or
-`Gradle <https://gradle.org/>`__, or set up a runnable project by
-following our Quick Start guide.
+MongoDB driver for synchronous Java applications.
+Follow the :ref:`Quick Start <java-sync-quickstart>` guide to learn how
+to install the driver by using `Maven <https://maven.apache.org/>`__ or
+`Gradle <https://gradle.org/>`__ in your project and set up a runnable app.
```

Comment:
Suggestion:

I think "install the driver" could be interpreted as launching an installer and/or having it available for anything, so the following might be more precise:

"learn how to add the driver to your project by using Maven..."

To make the items joined by the conjunction parallel, I think it could be good to include the prepositional phrase:

"and [how] to set up a runnable app"